cross/mosh/Makefile

```diff
@@ -0,0 +1,20 @@
+PKG_NAME = mosh
+PKG_VERS = 1.3.2
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/mobile-shell/mosh/releases/download/$(PKG_DIR)
+
+DEPENDS = cross/zlib cross/openssl cross/protobuf cross/ncurses
+#cross/perl
+
+HOMEPAGE = https://mosh.org
+COMMENT  = Mosh: the mobile shell
+LICENSE  = GPLv3+
+
+GNU_CONFIGURE = 1
+
+include ../../mk/spksrc.cross-cc.mk
+
+export PATH := $(PATH):$(STAGING_INSTALL_PREFIX)/bin
```

Comment:
What is reason to include these two lines ?

Reply:
They are needed to get `configure.sh` of mosh to pick up the `protobuf`exacutable. I'm not sure if there is a better way to get it working..